### PR TITLE
ADM-975 [frontend][backend]: show dora charts for each pipeline

### PR DESCRIPTION
**Pull request by zhou-yinyuan**
## Summary

...

## Before

_Description_

image

## After

_Description_

image

## Note

_Null_

### CODE REVIEW

File: frontend/src/containers/ReportStep/DoraMetricsChart/PipelineSelector/index.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+interface Props {
+  options: string[];
+  value: string;
+  onUpDatePipeline: (value: string) => void;
+  title: string;
+}
+
+export default function PipelineSelector({ options, value, onUpDatePipeline, title }: Props) {
+  const label = '';
```
Comment (guzhongren) [inline frontend/src/containers/ReportStep/DoraMetricsChart/PipelineSelector/index.tsx:16]: there is a `EMPTY` string in the constant.ts

**Review by guzhongren — APPROVED** [review]

**Comment by guzhongren** [comment]
能顺手给图表加个resize 么
ref to https://blog.csdn.net/qq6759/article/details/95632822

thx

File: frontend/src/containers/ReportStep/DoraMetricsChart/PipelineSelector/index.tsx (hunk 1 above)
Comment (zhou-yinyuan) [reply frontend/src/containers/ReportStep/DoraMetricsChart/PipelineSelector/index.tsx:16]: done

**Review by Mandy-Tang — APPROVED** [review]

**Comment by Mandy-Tang** [comment]
有一个sonar issue~

**Pull request merged**